### ISSUE TITLE
About data preparation

### ISSUE THREAD

**Issue by Randy-1009** [mxin262/SwinTextSpotter#7]
Does the annotation of data need to fix the number of annotation points? Or does it support any number of annotation points? Is there any special documentation on the data annotation specification? Thank you very much~

**Comment by mxin262** [comment]
Yes, you are right. It support any number of annotation points. There is a [example scripts](https://drive.google.com/file/d/1XgI6sZAlnrNCB24Rm3OW_FULp2RhsKUs/view) to generate the data annotation.

**Comment by Randy-1009** [comment]
> 

Thank you for your answer~
